### PR TITLE
Fixing dask test for presubmits

### PR DESCRIPTION
**Pull request by prince-cs**

### CODE REVIEW

File: dask/verify_dask_standalone.py [hunk 1]
```diff
@@ -1,11 +1,6 @@
-from dask.distributed import Client
 import dask.array as da
-import dask_ml
-import dask_bigquery
 
 import numpy as np
 
-client = Client("localhost:8786")
```
Comment (cjac) [inline dask/verify_dask_standalone.py:8]: Can you convince me that we don't care about dask.distributed ?

File: dask/verify_dask_yarn.py [hunk 2]
```diff
@@ -1,13 +1,6 @@
-from dask_yarn import YarnCluster
-from dask.distributed import Client
 import dask.array as da
-import dask_ml
-import dask_bigquery
 
 import numpy as np
 
-cluster = YarnCluster()
```
Comment (cjac) [inline dask/verify_dask_yarn.py:9]: This is the crux of the test.  We can't remove it and still call this a test.

**Review by cjac — COMMENTED** [review]
Please ensure that the tests pass

File: dask/verify_dask_standalone.py (hunk 1 above)
Comment (prince-cs) [reply dask/verify_dask_standalone.py:8]: This variable is not being used in the script. It is only declared.

File: dask/verify_dask_yarn.py (hunk 2 above)
Comment (prince-cs) [reply dask/verify_dask_yarn.py:9]: This variable is not being used in the script. It is only declared.

File: dask/verify_dask_standalone.py (hunk 1 above)
Comment (cjac) [reply dask/verify_dask_standalone.py:8]: The process of instantiating the object is the test.  If an exception is thrown, the test is a FAIL as it should be

File: dask/verify_dask_yarn.py (hunk 2 above)
Comment (cjac) [reply dask/verify_dask_yarn.py:9]: That's not a declaration, that's an instantiation

**Pull request closed without merging**